rfc/22/README.md

```diff
@@ -0,0 +1,145 @@
+---
+domain: github.com
+shortname: 22/MPAUX
+name: Multipool Auxiliary Layout Parameters Generation
+status: raw
+editors: Suvrat Joshi <suvrat.joshi@seagate.com>
+---
+
+## Language
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+## Abstract
+
+The multipools feature gives cluster an ability to have multiple pools (sets) of the disk/storage resources and IO's can be done on the desired pool.
+The auxiliary pools feature is required for creating auxialiry pool sets, one of which could be used upon failure of one or more disk resources.
+
+## Design
+
+If some disk resources in a cluster fail for some reason temporarily (because of reasons like reboot or transient network error etc.), then it is expected that cluster still works properly with reduced pool layout for failure handling. Let's say the initial pool layout configuration is 4+2, and a storage set has 6 data volumes. If 1-2 of those data volumes fail, the pool is switched to 2+2 layout on the remaining data volumes. There can be several combinations of the 2+2 data volume layouts, depending on which particular two volumes fail. All such combinations need to be pre-populated (auto-generated by Hare from the CDF configuration) by the hctl bootstrap script.
```

Comment:
[* suggestion * ] Following example presents a typical Base pool, auxiliary pool and their typical data distribution attributes,